ios/FluentUI/Calendar/Views/CalendarViewDayCell.swift

```diff
@@ -187,21 +198,16 @@ class CalendarViewDayCell: UICollectionViewCell {
     private func updateViews() {
         switch textStyle {
         case .primary:
-            dateLabel.textColor = Colors.Calendar.Day.textPrimary
+            dateLabel.textColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground3])
         case .secondary:
-            dateLabel.textColor = Colors.Calendar.Day.textSecondary
+            dateLabel.textColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground1])
         }
 
-        switch backgroundStyle {
```

Comment:
with this removal, is there any usage of CalendarViewDayCellBackgroundStyle ?

Reply:
There's no needed usage for it anymore. I'll ahead and remove it.